lib/matplotlib/animation.py

```diff
@@ -269,18 +263,6 @@ def frame_size(self):
         w, h = self.fig.get_size_inches()
         return int(w * self.dpi), int(h * self.dpi)
 
-    def _adjust_frame_size(self):
-        if self.codec == 'h264':
-            wo, ho = self.fig.get_size_inches()
-            w, h = adjusted_figsize(wo, ho, self.dpi, 2)
-            if not (wo, ho) == (w, h):
-                self.fig.set_size_inches(w, h, forward=True)
```

Comment:
The fix might be to set `forward=False` here.  Some of the backends will feed-back and _mostly_ go to the size you want (which is another set of things we need to fix, but).